src/main/java/com/ericsson/ei/frontend/BackEndInformationController.java

```diff
@@ -42,15 +42,15 @@
     @CrossOrigin
     @RequestMapping(value = "/backend", method = RequestMethod.GET)
     public ResponseEntity<String> getInstances(Model model, HttpServletRequest request) {
-        LOG.debug("Recieved request for instances.");
+        LOG.debug("Recieved request for retrieving EI Backend instances list.");
         ResponseEntity<String> response = backEndInfoContUtils.handleRequestForInstances(request);
         return response;
     }
 
     @CrossOrigin
     @RequestMapping(value = "/backend", method = RequestMethod.PUT)
     public ResponseEntity<String> switchBackEndInstance(Model model, HttpServletRequest request) {
-        LOG.debug("Recieved request to switch back end.");
+        LOG.debug("Recieved request to switch EI back end instance.");
```

Comment:
Please change "back end" to "backend" everywhere so it's consistent in both logging and user documentation.

Reply:
You're right, I took a closer look at the documentation and we decided to use back-end and front-end when we refer to the nouns. But my issue here is that the comments in the code was using different words. Please use the hyphened words from now on.

Reply:
I think "backend" term is commonly used in a technical world. I use always "backend" term when I write the word in code and technical comments and emails. Easier to read and spell the word when its not decoupled, thats my opinion. But the documentation on github io pages can maybe use "back-end" or "back end" format.

Reply:
I agree the "backend" is easier to write, and I would prefer using this only.  I'll approve this PR. I made a PR #203 to fix the markdown documentation that I found, and see what @a-breid-ericsson thinks about it :)

Reply:
Yes, I have added my opinions in that PR as well :)